ruleset/rules/0016-wazuh_rules.xml

```diff
@@ -154,28 +154,28 @@
   <rule id="231" level="7">
     <if_sid>230</if_sid>
     <field name="alert_type">80_percentage</field>
-    <description>The file limit set for this agent is $(db_entry_file_limit). Now, $(file_count) files are being monitored and all files that come after the limit will not be monitored. Change this setting in centralized configuration or locally on the agent.</description>
+    <description>The limit of entries of fim DB table $(fim_db_table) is close to being reached. At this moment there are $(entries_count) entries and the limit is $(entries_limit). If the limit is reached some events can be lost. You can modify this setting in the centralized configuration or locally in the agent.</description>
```

Comment:
This description is a bit technical for an alert description, for example, the mention of the `fim DB table`. Try to review the rules description to make them more user-readable.

Reply:
done